2019-01-09 09:41:06,973 [INFO] runcase.py line:116: From Excel import testsuite success
2019-01-09 09:42:02,999 [DEBUG] runcase.py line:147: testsuite has been parsed:
[{'id': 'login', 'title': '登录', 'condition': 'BASE', 'designer': '的的', 'flag': 'Y', 'result': None, 'remark': '', 'priority': 'M', 'steps': [{'control': '', 'no': '1', 'keyword': 'DRAG_AND_DROP', 'page': 'app相册', 'element': '相册完成', 'data': {'text': '33'}, 'expected': {}, 'output': {'text': '12'}, 'score': '', 'remark': '', '_keyword': 'DRAG_AND_DROP', '_element': '相册完成', '_data': '33', 'vdata': '33', '_expected': '', '_output': '12'}]}]
2019-01-09 09:42:11,797 [INFO] testcase.py line:57: ---------- Run the TestCase: login | 登录 -------------
2019-01-09 09:42:11,801 [INFO] testcase.py line:78: Run the Step: login - 1 | DRAG_AND_DROP | 相册完成
2019-01-09 09:42:12,268 [ERROR] testcase.py line:185: Run the Step: 1|DRAG_AND_DROP|app相册-相册完成 is Failure
Traceback (most recent call last):
  File "E:/py_workspaces/clearbug\sweetest\testcase.py", line 110, in run
    getattr(web, step['keyword'].lower())(step)
  File "E:/py_workspaces/clearbug\sweetest\keywords\web.py", line 236, in drag_and_drop
    source = locating_element(element[0])
  File "E:/py_workspaces/clearbug\sweetest\locator.py", line 28, in locating_element
    return _locating_element(element,element_wait_timeout)
  File "E:/py_workspaces/clearbug\sweetest\locator.py", line 44, in _locating_element
    if el['by'].lower() in ('title', 'url', 'current_url'):
TypeError: string indices must be integers
2019-01-09 09:42:12,317 [WARNING] testsuite.py line:128: Run the testcase: login|登录 Fail, BASE is not Pass. Break the AutoTest
2019-01-09 09:43:40,362 [INFO] runcase.py line:116: From Excel import testsuite success
2019-01-09 09:43:55,616 [DEBUG] runcase.py line:147: testsuite has been parsed:
[{'id': 'login', 'title': '登录', 'condition': 'BASE', 'designer': '的的', 'flag': 'Y', 'result': None, 'remark': '', 'priority': 'M', 'steps': [{'control': '', 'no': '1', 'keyword': 'DRAG_AND_DROP', 'page': 'app相册', 'element': '相册完成', 'data': {'text': '33'}, 'expected': {}, 'output': {'text': '12'}, 'score': '', 'remark': '', '_keyword': 'DRAG_AND_DROP', '_element': '相册完成', '_data': '33', 'vdata': '33', '_expected': '', '_output': '12'}]}]
2019-01-09 09:43:55,619 [INFO] testcase.py line:57: ---------- Run the TestCase: login | 登录 -------------
2019-01-09 09:43:55,623 [INFO] testcase.py line:78: Run the Step: login - 1 | DRAG_AND_DROP | 相册完成
2019-01-09 11:10:45,456 [INFO] runcase.py line:117: From Excel import testsuite success
2019-01-09 11:10:48,316 [DEBUG] runcase.py line:149: testsuite has been parsed:
[{'id': 'login', 'title': '登录', 'condition': 'BASE', 'designer': '的的', 'flag': 'N', 'result': None, 'remark': '', 'priority': 'M', 'steps': [{'control': '', 'no': '1', 'keyword': 'OPEN', 'page': '通用', 'element': 'url#<登录地址>', 'data': {'text': '3'}, 'expected': {}, 'output': {}, 'score': '', 'remark': '', '_keyword': 'OPEN', '_element': 'url#<登录地址>', '_data': '3', 'vdata': '3', '_expected': '', '_output': ''}]}]
2019-01-09 11:10:48,318 [INFO] testsuite.py line:98: Run the testcase: login|登录 Skipped, because the flag=N or the condition=snippet
2019-01-09 11:10:48,371 [ERROR] runcase.py line:170: *** Save the report is fail ***
Traceback (most recent call last):
  File "E:/py_workspaces/clearbug\sweetest\runcase.py", line 167, in run
    data = testsuite2data(testsuite)
  File "E:/py_workspaces/clearbug\sweetest\data.py", line 152, in testsuite2data
    result = [[g.header_custom[key.lower()] for key in header.values()]]
  File "E:/py_workspaces/clearbug\sweetest\data.py", line 152, in <listcomp>
    result = [[g.header_custom[key.lower()] for key in header.values()]]
AttributeError: 'Global' object has no attribute 'header_custom'
2019-01-09 17:46:37,721 [INFO] runcase.py line:118: From Excel import testsuite success
2019-01-09 17:46:40,488 [DEBUG] runcase.py line:150: testsuite has been parsed:
[{'id': 'login', 'title': '登录', 'condition': 'BASE', 'designer': '的的', 'flag': 'N', 'result': None, 'remark': '', 'priority': 'M', 'steps': [{'control': '', 'no': '1', 'keyword': 'OPEN', 'page': '通用', 'element': 'url#<登录地址>', 'data': {'text': '3'}, 'expected': {}, 'output': {}, 'score': '', 'remark': '', '_keyword': 'OPEN', '_element': 'url#<登录地址>', '_data': '3', 'vdata': '3', '_expected': '', '_output': ''}]}]
2019-01-09 17:46:40,489 [INFO] testsuite.py line:98: Run the testcase: login|登录 Skipped, because the flag=N or the condition=snippet
2019-01-09 17:46:40,568 [ERROR] runcase.py line:171: *** Save the report is fail ***
Traceback (most recent call last):
  File "E:/py_workspaces/clearbug\sweetest\runcase.py", line 168, in run
    data = testsuite2data(testsuite)
  File "E:/py_workspaces/clearbug\sweetest\data.py", line 127, in testsuite2data
    result = [[g.header_custom[key.lower()] for key in header.values()]]
  File "E:/py_workspaces/clearbug\sweetest\data.py", line 127, in <listcomp>
    result = [[g.header_custom[key.lower()] for key in header.values()]]
KeyError: 'step'
2019-01-09 17:48:01,678 [INFO] runcase.py line:118: From Excel import testsuite success
2019-01-09 17:48:04,551 [DEBUG] runcase.py line:150: testsuite has been parsed:
[{'id': 'login', 'title': '登录', 'condition': 'BASE', 'designer': '的的', 'flag': 'N', 'result': None, 'remark': '', 'priority': 'M', 'steps': [{'control': '', 'no': '1', 'keyword': 'OPEN', 'page': '通用', 'element': 'url#<登录地址>', 'data': {'text': '3'}, 'expected': {}, 'output': {}, 'score': '', 'remark': '', '_keyword': 'OPEN', '_element': 'url#<登录地址>', '_data': '3', 'vdata': '3', '_expected': '', '_output': ''}]}]
2019-01-09 17:48:04,553 [INFO] testsuite.py line:98: Run the testcase: login|登录 Skipped, because the flag=N or the condition=snippet
2019-01-09 17:55:31,961 [INFO] runcase.py line:118: From Excel import testsuite success
2019-01-09 17:55:34,709 [DEBUG] runcase.py line:150: testsuite has been parsed:
[{'id': 'login', 'title': '登录', 'condition': 'BASE', 'designer': '的的', 'flag': 'N', 'result': None, 'remark': '', 'priority': 'M', 'steps': [{'control': '', 'no': '1', 'keyword': 'OPEN', 'page': '通用', 'element': 'url#<登录地址>', 'data': {'text': '3'}, 'expected': {}, 'output': {}, 'score': '', 'remark': '', '_keyword': 'OPEN', '_element': 'url#<登录地址>', '_data': '3', 'vdata': '3', '_expected': '', '_output': ''}]}]
2019-01-09 17:55:34,710 [INFO] testsuite.py line:98: Run the testcase: login|登录 Skipped, because the flag=N or the condition=snippet
2019-01-09 17:55:52,019 [ERROR] runcase.py line:171: *** Save the report is fail ***
Traceback (most recent call last):
  File "E:/py_workspaces/clearbug\sweetest\runcase.py", line 168, in run
    data = testsuite2data(testsuite)
  File "E:/py_workspaces/clearbug\sweetest\data.py", line 127, in testsuite2data
    result = [[g.header_custom[key.lower()] for key in header.values()]]
  File "E:/py_workspaces/clearbug\sweetest\data.py", line 127, in <listcomp>
    result = [[g.header_custom[key.lower()] for key in header.values()]]
KeyError: 'step'
2019-01-09 17:56:06,803 [INFO] runcase.py line:118: From Excel import testsuite success
2019-01-09 17:56:09,427 [DEBUG] runcase.py line:150: testsuite has been parsed:
[{'id': 'login', 'title': '登录', 'condition': 'BASE', 'designer': '的的', 'flag': 'N', 'result': None, 'remark': '', 'priority': 'M', 'steps': [{'control': '', 'no': '1', 'keyword': 'OPEN', 'page': '通用', 'element': 'url#<登录地址>', 'data': {'text': '3'}, 'expected': {}, 'output': {}, 'score': '', 'remark': '', '_keyword': 'OPEN', '_element': 'url#<登录地址>', '_data': '3', 'vdata': '3', '_expected': '', '_output': ''}]}]
2019-01-09 17:56:09,428 [INFO] testsuite.py line:98: Run the testcase: login|登录 Skipped, because the flag=N or the condition=snippet
2019-01-09 17:56:09,473 [ERROR] runcase.py line:171: *** Save the report is fail ***
Traceback (most recent call last):
  File "E:/py_workspaces/clearbug\sweetest\runcase.py", line 168, in run
    data = testsuite2data(testsuite)
  File "E:/py_workspaces/clearbug\sweetest\data.py", line 127, in testsuite2data
    result = [[g.header_custom[key.lower()] for key in header.values()]]
  File "E:/py_workspaces/clearbug\sweetest\data.py", line 127, in <listcomp>
    result = [[g.header_custom[key.lower()] for key in header.values()]]
KeyError: 'step'
2019-01-09 18:08:53,364 [INFO] runcase.py line:118: From Excel import testsuite success
2019-01-09 18:08:56,098 [DEBUG] runcase.py line:150: testsuite has been parsed:
[{'id': 'login', 'title': '登录', 'condition': 'BASE', 'designer': '的的', 'flag': 'N', 'result': None, 'remark': '', 'priority': 'M', 'steps': [{'control': '', 'no': '1', 'keyword': 'OPEN', 'page': '通用', 'element': 'url#<登录地址>', 'data': {'text': '3'}, 'expected': {}, 'output': {}, 'score': '', 'remark': '', '_keyword': 'OPEN', '_element': 'url#<登录地址>', '_data': '3', 'vdata': '3', '_expected': '', '_output': ''}]}]
2019-01-09 18:08:56,099 [INFO] testsuite.py line:98: Run the testcase: login|登录 Skipped, because the flag=N or the condition=snippet
2019-01-09 18:14:06,209 [INFO] runcase.py line:118: From Excel import testsuite success
2019-01-09 18:14:08,923 [DEBUG] runcase.py line:150: testsuite has been parsed:
[{'id': 'login', 'title': '登录', 'condition': 'BASE', 'designer': '的的', 'flag': 'N', 'result': None, 'remark': '', 'priority': 'M', 'steps': [{'control': '', 'no': '1', 'keyword': 'OPEN', 'page': '通用', 'element': 'url#<登录地址>', 'data': {'text': '3'}, 'expected': {}, 'output': {}, 'score': '', 'remark': '', '_keyword': 'OPEN', '_element': 'url#<登录地址>', '_data': '3', 'vdata': '3', '_expected': '', '_output': ''}]}]
2019-01-09 18:14:08,924 [INFO] testsuite.py line:98: Run the testcase: login|登录 Skipped, because the flag=N or the condition=snippet
2019-01-09 18:14:16,401 [ERROR] runcase.py line:171: *** Save the report is fail ***
Traceback (most recent call last):
  File "E:/py_workspaces/clearbug\sweetest\runcase.py", line 168, in run
    data = testsuite2data(testsuite)
  File "E:/py_workspaces/clearbug\sweetest\data.py", line 127, in testsuite2data
    result = [[g.header_custom[key.lower()] for key in header.values()]]
  File "E:/py_workspaces/clearbug\sweetest\data.py", line 127, in <listcomp>
    result = [[g.header_custom[key.lower()] for key in header.values()]]
KeyError: 'keyword'
2019-01-09 18:18:30,755 [ERROR] runcase.py line:120: *** From Excel import testsuite fail ***
Traceback (most recent call last):
  File "E:/py_workspaces/clearbug\sweetest\runcase.py", line 115, in run
    set_g_header(testsuite)
  File "E:/py_workspaces/clearbug\sweetest\utility.py", line 116, in set_g_header
    for sk,sv in testcases[0].get("steps").items():
AttributeError: 'list' object has no attribute 'items'
2019-01-09 18:18:46,704 [ERROR] runcase.py line:120: *** From Excel import testsuite fail ***
Traceback (most recent call last):
  File "E:/py_workspaces/clearbug\sweetest\runcase.py", line 115, in run
    set_g_header(testsuite)
  File "E:/py_workspaces/clearbug\sweetest\utility.py", line 116, in set_g_header
    for sk,sv in testcases[0].get("steps").items():
AttributeError: 'list' object has no attribute 'items'
2019-01-09 18:22:05,451 [INFO] runcase.py line:118: From Excel import testsuite success
2019-01-09 18:22:08,158 [DEBUG] runcase.py line:150: testsuite has been parsed:
[{'id': 'login', 'title': '登录', 'condition': 'BASE', 'designer': '的的', 'flag': 'N', 'result': None, 'remark': '', 'priority': 'M', 'steps': [{'control': '', 'no': '1', 'keyword': 'OPEN', 'page': '通用', 'element': 'url#<登录地址>', 'data': {'text': '3'}, 'expected': {}, 'output': {}, 'score': '', 'remark': '', '_keyword': 'OPEN', '_element': 'url#<登录地址>', '_data': '3', 'vdata': '3', '_expected': '', '_output': ''}]}]
2019-01-09 18:22:08,160 [INFO] testsuite.py line:98: Run the testcase: login|登录 Skipped, because the flag=N or the condition=snippet
2019-01-09 18:27:33,046 [INFO] runcase.py line:118: From Excel import testsuite success
2019-01-09 18:27:35,783 [DEBUG] runcase.py line:150: testsuite has been parsed:
[{'id': 'login', 'title': '登录', 'condition': 'BASE', 'designer': '的的', 'flag': 'Y', 'result': None, 'remark': '', 'priority': 'M', 'steps': [{'control': '', 'no': '1', 'keyword': 'OPEN', 'page': '通用', 'element': 'url#<登录地址>', 'data': {'text': '3'}, 'expected': {}, 'output': {}, 'score': '', 'remark': '', '_keyword': 'OPEN', '_element': 'url#<登录地址>', '_data': '3', 'vdata': '3', '_expected': '', '_output': ''}]}]
2019-01-09 18:27:35,784 [INFO] testcase.py line:57: ---------- Run the TestCase: login | 登录 -------------
2019-01-09 18:27:35,784 [INFO] testcase.py line:78: Run the Step: login - 1 | OPEN | url#<登录地址>
2019-01-09 18:27:36,162 [ERROR] testcase.py line:185: Run the Step: 1|OPEN|通用-url#v3-yfb-portal.hd-cloud.com is Failure
Traceback (most recent call last):
  File "E:/py_workspaces/clearbug\sweetest\testcase.py", line 110, in run
    getattr(web, step['keyword'].lower())(step)
  File "E:/py_workspaces/clearbug\sweetest\keywords\web.py", line 58, in open
    g.driver.get(value)
  File "E:\py_workspaces\clearbug\venv\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 333, in get
    self.execute(Command.GET, {'url': url})
  File "E:\py_workspaces\clearbug\venv\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "E:\py_workspaces\clearbug\venv\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"Cannot navigate to invalid URL"}
  (Session info: chrome=71.0.3578.98)
  (Driver info: chromedriver=2.43.600210 (68dcf5eebde37173d4027fa8635e332711d2874a),platform=Windows NT 6.1.7601 SP1 x86_64)

2019-01-09 18:27:36,235 [WARNING] testsuite.py line:128: Run the testcase: login|登录 Fail, BASE is not Pass. Break the AutoTest
2019-01-09 18:33:22,639 [INFO] runcase.py line:118: From Excel import testsuite success
2019-01-09 18:33:25,487 [DEBUG] runcase.py line:150: testsuite has been parsed:
[{'id': 'login', 'title': '登录', 'condition': 'BASE', 'designer': '的的', 'flag': 'Y', 'result': None, 'remark': '', 'priority': 'M', 'steps': [{'control': '', 'no': '1', 'keyword': 'OPEN', 'page': '通用', 'element': 'url#<登录地址>', 'data': {'text': '3'}, 'expected': {}, 'output': {}, 'score': '', 'remark': '', '_keyword': 'OPEN', '_element': 'url#<登录地址>', '_data': '3', 'vdata': '3', '_expected': '', '_output': ''}]}]
2019-01-09 18:33:25,488 [INFO] testcase.py line:57: ---------- Run the TestCase: login | 登录 -------------
2019-01-09 18:33:25,489 [INFO] testcase.py line:78: Run the Step: login - 1 | OPEN | url#<登录地址>
